site/content/get-started.md

```diff
@@ -0,0 +1,311 @@
+tea-7b7d6c947d-8xmhm      1/1     Running   0          80s
+```
+
+---
+
+### Create Gateway and HTTPRoute resources
+
+Create the file _gateway.yaml_ with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/gateway.yaml">}}
+
+Apply it using `kubectl`:
+
+```shell
+kubectl apply -f gateway.yaml
+```
+```text
+gateway.gateway.networking.k8s.io/gateway created
+```
+
+Create the file _cafe-routes.yaml_ with the following contents:
+
+{{< ghcode "https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/refs/heads/main/examples/cafe-example/cafe-routes.yaml">}}
+
+Apply it using `kubectl`:
+
+```shell
+kubectl apply -f cafe-routes.yaml
+```
+```text
+httproute.gateway.networking.k8s.io/coffee created
+httproute.gateway.networking.k8s.io/tea created
+```
+
+You can check that all of the expected services are available using `kubectl get`:
+
+```shell
+kubectl get service --all-namespaces
+```
+```text
+NAMESPACE       NAME            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
+default         coffee          ClusterIP   10.96.18.163    <none>        80/TCP                       2m51s
+default         kubernetes      ClusterIP   10.96.0.1       <none>        443/TCP                      4m41s
+default         tea             ClusterIP   10.96.169.132   <none>        80/TCP                       2m51s
+kube-system     kube-dns        ClusterIP   10.96.0.10      <none>        53/UDP,53/TCP,9153/TCP       4m40s
+nginx-gateway   nginx-gateway   NodePort    10.96.186.45    <none>        80:31437/TCP,443:31438/TCP   3m6s
+```
+
+---
+
+## Test NGINX Gateway Fabric
+
+The cluster was configured with port `8080` as the `containerPort` value, alongside the `nodePort` value of the NodePort service.
+
+Since the NodePort `targetPort` values match the _tea_ and _coffee_ service `port` values, no port forwarding is required.
+
+You can use `curl` to test the new services by targeting the hostname (_cafe.example.com_) with the _/coffee_ and _/tea_ paths:
+
+```shell
+curl --resolve cafe.example.com:8080:127.0.0.1 http://cafe.example.com:8080/coffee
+```
+```text
+Server address: 10.244.0.6:8080
+Server name: coffee-6db967495b-984mx
+Date: 17/Oct/2024:15:50:22 +0000
+URI: /coffee
+Request ID: 8ad83b06ea42b996ad6bd28032b38e28
+```
+
+```shell
+curl --resolve cafe.example.com:8080:127.0.0.1 http://cafe.example.com:8080/coffee
+```
+```text
+Server address: 10.244.0.6:8080
+Server name: coffee-6db967495b-984mx
+Date: 17/Oct/2024:15:50:29 +0000
+URI: /coffee
+Request ID: 2dfc85564dd5b5dad7e62b980bb60ee1
+```
+
+---
+
+## Additional reading
+
+- [Install NGINX Gateway Fabric](http://localhost:1313/nginx-gateway-fabric/installation/installing-ngf/), for additional ways to install NGINX Gateway Fabric
+- [How-to guides](http://localhost:1313/nginx-gateway-fabric/how-to/), for configuring your cluster
```

Comment:
Are these supposed to be linked to localhost?

Reply:
Nope, I lost the plot by the end of this document.